Foundation/Data.swift

```diff
@@ -12,7 +12,7 @@
 
 #if DEPLOYMENT_RUNTIME_SWIFT
 
-#if os(Linux)
+#if os(Linux) || os(Windows)
```

Comment:
Would this be better as a `#if !canImport(Darwin)` as I dont think it is supported on any other platforms apart from Apple's?

Reply:
That is true. The only other malloc implementation that has this feature iirc is jemalloc.

Reply:
It would be interesting to be able to use jemalloc on other targets.  I know that there are environments running off of jemalloc.  I don't think that we have a good way to identify that yet.  Perhaps having a macro from the build system `-DHAVE_MALLOC_GOOD_SIZE`?